src/averageAges.js

```diff
@@ -14,12 +14,19 @@
  *
  * @return {number}
  */
+
+function calculateAverageAge(fiteredList) {
+  return fiteredList.reduce((sum, person) =>
+    sum + (person.died - person.born), 0) / fiteredList.length;
```

Comment:
```suggestion
  return fiteredList.reduce((sum, person) => (
    sum + (person.died - person.born)
  ), 0) / fiteredList.length;
```

Reply:
Yep, 2.[CODE STYLE]. 
The code was too attractive in its compact form. And I accidentally missed it like that